### PR TITLE
Add additional cgroup docs

### PR DESCRIPTION
**Pull request by andrewkroh**
- Document boolean cgroup parameter.
- Describe how to run Metricbeat in a container.
- Set kibana types for cgroup files.

### CODE REVIEW

File: metricbeat/docs/metricbeat-in-a-container.asciidoc [hunk 1]
```diff
@@ -0,0 +1,73 @@
+== Running Metricbeat in a Container
```
Comment (ruflin) [inline metricbeat/docs/metricbeat-in-a-container.asciidoc]: @andrewkroh This should be reused as a blog post. Great stuff.

File: metricbeat/docs/metricbeat-in-a-container.asciidoc [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+---
+
+<1> Metricbeat's system module collects much of its data through the Linux proc
+filesystem which is normally located at `/proc`. Due to the fact that containers
+are isolated as much as possible from the host, the data inside of the
+containers `/proc` is different than the host's `/proc`. To account for this you
+can mount the host's `/proc` filesystem inside of the container and tell
+Metricbeat to look inside the `/hostfs` directory when looking for `/proc` by
+using the `-system.hostfs=/hostfs` CLI flag.
+<2> If you have enabled cgroup reporting from the system process metricset then
```
Comment (ruflin) [inline metricbeat/docs/metricbeat-in-a-container.asciidoc]: Perhaps quickly mention that it is experimental?

File: metricbeat/docs/metricbeat-in-a-container.asciidoc [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+<3> If you want to be able to monitor filesystems from the host using the
+system filesystem metricset then those filesystems need to be mounted inside
+of the container. They can be mounted at any location.
+<4> The system network metricset uses data from `/proc/net/dev`, or
+`/hostfs/proc/net/dev` when using `-system.hostfs=/hostfs`, and the only way
+to make this file contain the host's network devices is to use the `--net=host`
+flag. This is due to Linux namespacing, simply bind mounting the host's `/proc`
+to `/hostfs/proc` is not sufficient.
+
+Next let's look at an example of monitoring a containerized service from a
```
Comment (ruflin) [inline metricbeat/docs/metricbeat-in-a-container.asciidoc]: Perhaps add a titel here to make it clear, these are two separate topics. First monitor host system, second monitor service.

File: metricbeat/docs/metricbeat-in-a-container.asciidoc (hunk 2 above)
Comment (andrewkroh) [inline metricbeat/docs/metricbeat-in-a-container.asciidoc]: Added

File: metricbeat/docs/metricbeat-in-a-container.asciidoc (hunk 3 above)
Comment (andrewkroh) [inline metricbeat/docs/metricbeat-in-a-container.asciidoc]: Added

**Pull request merged**